app/components/AddToCalendar/AddToCalendar.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button, Card, Icon } from '@webkom/lego-bricks';
+import { useState } from 'react';
+import {
+  getIcalUrl,
+  getIcalUrlGoogle,
+} from 'app/routes/events/components/EventFooter';
+import styles from './AddToCalendar.css';
+import type { Meeting } from 'app/models';
```

Comment:
```suggestion
import type { DetailedMeeting } from 'app/store/models/Meeting';
```
The types in `app/models` are outdated